### ISSUE TITLE
Signature of unnamed POIs

### ISSUE THREAD

**Issue by Zavgor1990** [organicmaps/organicmaps#4453]
If the point has an address specified, but the name tag is not specified, the house number is displayed in the signature. The problem is inherited from Maps.me .
![IMG_20230212_112801](https://user-images.githubusercontent.com/124589977/218300691-e3cdbd39-a655-4966-a8d7-e2c8519a588c.jpg)

**Comment by Jean-BaptisteC** [comment]
I think priorities to show POI name is incorrect
If name of POI is not specified please add this name on editor and push change on OpenStreetMap

**Comment by biodranik** [comment]
It's our bug. @vng

**Comment by vng** [comment]
Now this is a common logic for all features (POIs, Buildings, Addresses, Entrances).
name -> ref -> house number

We can try to distinguish "POI/non-POI" in code now, but better to improve style's definitions what to show in caption, imo.

**Comment by biodranik** [comment]
Does any other app draw a plain house number of the address on any POI? That looks _unusual_ and _unexpected_.

**Comment by vng** [comment]
Moreover most popular app doesn't draw house numbers at all. Never and nowhere :)

**Comment by Zavgor1990** [comment]
> Does any other app draw a plain house number of the address on any POI? That looks _unusual_ and _unexpected_.

This trouble with OrganicMaps, Maps.me, but Osmand hasn't it

**Comment by eisams** [comment]
> Now this is a common logic for all features (POIs, Buildings, Addresses, Entrances). name -> ref -> house number
> 
> We can try to distinguish "POI/non-POI" in code now, but better to improve style's definitions what to show in caption, imo.

Is the ref supposed to be rendered? At least this building does not get its ref rendered in OM. The ref is the building letter the hotel use to direct guests to the correct building

https://www.openstreetmap.org/way/256588556

**Comment by vng** [comment]
Yes, ref is rendered as regular name, but it is fetched from OSM for particular types only:
https://github.com/organicmaps/organicmaps/blob/master/generator/feature_builder.cpp#L226

**Comment by biodranik** [comment]
@vng we need to extend that list )
